clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -105,6 +105,8 @@ public final class ConsumerCoordinator extends AbstractCoordinator {
     private final boolean autoCommitEnabled;
     private final int autoCommitIntervalMs;
     private final ConsumerInterceptors<?, ?> interceptors;
+    // package private for testing
+    final AtomicInteger inFlightAsyncCommits;
     private final AtomicInteger pendingAsyncCommits;
```

Comment:
Could we add a comment for these two variables? I think that we were all confused by them while looking at this PR so adding clarifying comments would make sense here.

Reply:
I took the description of pendingAyncCommits from your comment https://github.com/apache/kafka/pull/13678#discussion_r1191927989, I hope I did that correctly.

Changed in: https://github.com/apache/kafka/pull/13678/commits/2bfedf0ec27f9d386d9cddb2ba24d53e533ab51f